src/BottomNavigation/BottomNavigation.js

```diff
@@ -64,6 +64,7 @@ function BottomNavigation(props: ProvidedProps & Props) {
   const className = classNames(classes.root, classNameProp);
 
   const children = React.Children.map(childrenProp, (child, childIndex) => {
+    if (child === null) return null;
```

Comment:
What about using `React.Children.isValidElement` instead?

Reply:
You meant `if (React.isValidElement(child)) return null;` ?
Yes, that would be even better